recipes/opencv/4.x/conanfile.py

```diff
@@ -25,6 +25,7 @@ class OpenCVConan(ConanFile):
     options = {
         "shared": [True, False],
         "fPIC": [True, False],
+        "build_world":[True, False],
```

Comment:
It's not a very good name like all recipe options `build_<something>`, I think`world` is a better name and consistent with other option `dnn` to build opencv_dnn.

Reply:
Agree, I will changed it.